tests/data/ligands_40__first-two-ligs.sdf

```diff
@@ -1,169 +0,0 @@
-338
```

Comment:
it looks like this is still used in a few places in `docking/`:

```
docking/dock_and_equilibrate.py
237:        default="tests/data/ligands_40__first-two-ligs.sdf",

docking/rigorous_work.py
427:        default="tests/data/ligands_40__first-two-ligs.sdf",
```

Reply:
Good catch